src/Akka.Streams.Kafka/Settings/ConsumerSettings.cs

```diff
@@ -246,7 +246,10 @@ public sealed class ConsumerSettings<TKey, TValue>
         public ConsumerSettings<TKey, TValue> WithProperty(string key, string value) =>
             Copy(properties: Properties.SetItem(key, value));
 
-        public ConsumerSettings<TKey, TValue> WithProperties(IDictionary<string, string> properties)
+        public ConsumerSettings<TKey, TValue> WithConsumerConfig(ConsumerConfig config)
```

Comment:
Convenience method to pass `Confluent.Kafka.ConsumerConfig` directly into our `ConsumerSettings`

Reply:
Nice - I think we have an open issue for this somewhere. We should mark that as closed once we get this merged.